go/protocol/chat1/notify.go

```diff
@@ -215,6 +215,11 @@ type ChatTLFFinalizeArg struct {
 	FinalizeInfo ConversationFinalizeInfo `codec:"finalizeInfo" json:"finalizeInfo"`
 }
 
+type ChatTLFResolveArg struct {
+	Uid    keybase1.UID   `codec:"uid" json:"uid"`
+	ConvID ConversationID `codec:"convID" json:"convID"`
```

Comment:
They would care about the fact that it happened, so they can refresh the stale name. But you're right the service can get it, I'll add that,